source/addonStore/dataManager.py

```diff
@@ -0,0 +1,227 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+import dataclasses
+import enum
+import json
+import os
+import pathlib
+from concurrent.futures import (
+	Future,
+	ThreadPoolExecutor
+)
+from datetime import datetime, timedelta
+
+from logHandler import log
+import requests
+
+import addonAPIVersion
+import typing
+from typing import (
+	Optional,
+	List,
+	Dict,
+	Callable,
+	Tuple,
+)
+from .models import (
+	_createModelFromData,
+	AddonDetailsModel,
+)
+
+
+class Channel(str, enum.Enum):
+	STABLE = "stable"
+	BETA = "beta"
+	ALL = "all"
+
+
+def _getCurrentApiVersionForURL() -> str:
+	currentVersion = addonAPIVersion.CURRENT
+	year, major, minor = currentVersion
+	return f"{year}.{major}.{minor}"
+
+
+def _getAddonStoreURL(channel: Channel, lang: str, nvdaApiVersion: str) -> str:
+	_baseURL = "https://nvaccess.org/addonStore/"

```

Comment:
Can this be a constant?

Reply:
I assume you mean module level constant? There would need to be a good rational to reduce the encapsulation.

Reply:
As long as there's only one function that needs the base url, I guess it is ok as is.